src/main/java/ch/njol/skript/expressions/ExprClicked.java

```diff
@@ -58,33 +64,96 @@
 @Since("1.0")
 @Events("click")
```

Comment:
You could mention the inventory click event here, too.

Reply:
```
clear saturation of player
remove all the saturation from the player
reset saturation of the player
```
?

What are these reviews

Reply:
what does this have to do with what I requested here?